examples/simple/main.tf

```diff
@@ -113,6 +121,7 @@ module "ses_domain" {
 
   receive_s3_bucket = module.ses_bucket.id
   receive_s3_prefix = local.ses_bucket_prefix
+  enable_spf_record = false
```

Comment:
Can you make this variable configurable to terratest so we can have 1 test with it enabled and a second test with it disabled?